README.md

```diff
@@ -1,61 +1,95 @@
+executed. Note however, that independent steps are executed concurrently with
+each other. Steps are often used for tasks that produce a result that is
+needed by their dependents such as a download, creation of a database index
+or training of a machine learning model.
+
+The End-to-End tests of QLever [example](./examples/qlever_e2e) demonstrates
+the usage and interaction of both container types.
+
 ## Differences between `docker-compose` and `gantry`
 
-- Additional `steps` which can be used for pipelines as they are sequentially
-  executed as soon as all required prevous steps are executed instead of all at
-  the same time as `services` in `docker-compose`. Steps and services can depend
-  on one another. This allows steps to use deployed services. To enable this
-  explicit sequentiality the `after` relation is introduced in `steps`.
 - Generating `.dot` file showing dependencies using `gantry dot`.
+  This is used for the illustration of the [examples](./examples).
 
-## Build/Download
-Binary releases are provided
-[here](https://github.com/ad-freiburg/gantry/releases). Alternatively you can
-build your own release as described in the *Building a Release* section.
+- Gantry only supports the list with equal-sings for
+[`environment`](https://docs.docker.com/compose/compose-file/#environment)
 
-Gantry is go getable through
+## Installation
 
-    go get github.com/ad-freiburg/gantry/cmd/gantry
+### Download a prebuild Release
 
-This will result in a binary without a set version.
+Binary releases are provided as
+[github releases](https://github.com/ad-freiburg/gantry/releases).
 
-## Building a Release
-To build a release version first make sure everything works, then edit the
-[Setup](#Setup) section of this Readme so the download link points to the
-future version. *Only after committing this final change tag the release*
+    cd /tmp
+    rm gantry_$(uname -m).tar.bz2
+    wget https://github.com/ad-freiburg/gantry/releases/download/v0.3.0/gantry_$(uname -m).tar.bz2
+    tar -xavf gantry_$(uname -m).tar.bz2
+    sudo mv gantry_$(uname -m)/gantry /usr/local/bin/gantry
 
-    git tag -a vX.Y.Z -m <message>
+    gantry --version
 
-Then build with `-ldflags` such that the version is added to the binary
+### Arch Linux
 
-    go build -ldflags="-X github.com/ad-freiburg/gantry.Version=$(git describe --always --long --dirty)" cmd/gantry/gantry.go
+Gantry is available in the [AUR](https://aur.archlinux.org/) as
+[gantry](https://aur.archlinux.org/packages/gantry) and
+[gantry-git](https://aur.archlinux.org/packages/gantry-git)
 
-Finally use the GitHub Releases mechanism to release a new version
+### From source
 
-## Installing from source
 To install gantry into the users ~/go/bin path it is enough to just run
 
     go install -ldflags="-X github.com/ad-freiburg/gantry.Version=$(git describe --always --long --dirty)" ./...
 
+To install gantry global copy it from ~/go/bin path or use
```

Comment:
globally

Reply:
Thanks, that's the word I thought I used.